hashicorp/react-components
# Enable `StandaloneLink` in `Actions`

🎟️ [Asana Task](https://app.asana.com/0/1202802428287999/1203018584186578/f)
🔍 [Preview Link](https://react-components-git-nels-introadd-standalone-fd8f07-hashicorp.vercel.app/components/actions)
🎨 [Figma](https://www.figma.com/file/7cYgDM618stjYUHDqAfRec/Components?node-id=572%3A1058)

---



## Description

This PR adds support for the `StandaloneLink` in `Actions`. The requirements are as follows:
- `Actions` supports all of the `Button` / `StandaloneLink` combinations in the [spec](https://www.figma.com/file/7cYgDM618stjYUHDqAfRec/branch/5GbGjYszP8XpIebLm64Sa9/Components?node-id=572%3A1058).
- The first CTA and second CTA are always the `primary` variation and `secondary` variation respectively.
- When both a `Button` and `StandaloneLink` render, the `StandaloneLink` always comes second.
Addition

The changeset includes the following:
- Adds `StandaloneLink` to the render method of `Actions`
- Changes the `url` prop for `Actions` to `href`. This was done to match the `<a>` attribute name.
- Removes the now cruft `variant` prop from CTAs in `Actions`
- Updates the docs and tests to account for the new variations
- Updates failing tests for `Intro` and `NextSteps`. This occurs due to the change of the `url` prop to `href`

## Validation Steps
- Navigate to [`Actions` in the preview](https://react-components-git-nels-introadd-standalone-fd8f07-hashicorp.vercel.app/components/actions)
- Under examples, verify that all of the variations match the Figma (see above)

### PR Checklist 🚀

Items in this checklist may not may not apply to your PR, but please consider each item carefully.
